environments/india/terraform.yml

```diff
@@ -318,6 +318,26 @@ servers:
     group: "rabbitmq"
     os: ubuntu_pro_bionic
 
+  - server_name: "rabbit_a4-india"
+    server_instance_type: t3.small
+    network_tier: "db-private"
+    az: "a"
+    volume_size: 80
```

Comment:
Current disk usage on existing machines is less than 15GB, can you lower this to 30 for both?